README.md

```diff
@@ -461,8 +461,6 @@ Marek Majkowski](https://blog.cloudflare.com/a-gentle-introduction-to-linux-kern
 
 [2019: "Hourglass	Fuzz:	A	Quick	Bug	Hunting	Method"](https://conference.hitb.org/hitbsecconf2019ams/materials/D1T2%20-%20Hourglass%20Fuzz%20-%20A%20Quick%20Bug%20Hunting%20Method%20-%20Moony%20Li,%20Todd%20Han,%20Lance%20Jiang%20&%20Lilang%20Wu.pdf) [slides]
 
-[2019: "Fuzzing File Systems via Two-Dimensional Input Space Exploration"](https://taesoo.kim/pubs/2019/xu:janus.pdf) [paper]
```

Comment:
Why did you remove this?

Reply:
2019: "Razzer: Finding Kernel Race Bugs through Fuzzing" [video]

**2019: "Fuzzing File Systems via Two-Dimensional Input Space Exploration" [paper]**

2019: "PeriScope: An Effective Probing and Fuzzing Framework for the Hardware-OS Boundary" [paper]

2019: "Hourglass Fuzz: A Quick Bug Hunting Method" [slides]

**2019: "Fuzzing File Systems via Two-Dimensional Input Space Exploration" [paper]**

2018: "RAZZER: Finding Kernel Race Bugs through Fuzzing" [paper]

There are two same lines.